scripts/uosc.lua

```diff
@@ -4064,14 +4064,20 @@ if options.pause_on_click_shorter_than > 0 then
 end
 mp.set_key_bindings(mouse_keybinds, 'mouse_movement', 'force')
 mp.enable_key_bindings('mouse_movement', 'allow-vo-dragging+allow-hide-cursor')
-
-mp.register_event('file-loaded', function()
-	parse_chapters()
-	local title_template = mp.get_property_native('title')
+local function update_title(title_template)
 	if title_template:sub(-6) == ' - mpv' then title_template = title_template:sub(1, -7) end
 	set_state('title', mp.command_native({"expand-text", title_template}))
+end
+mp.register_event('file-loaded', function()
+	parse_chapters()
+	update_title(mp.get_property_native('title'))
 end)
 mp.register_event('end-file ', function() set_state('title', nil) end)
+mp.observe_property('title', 'string', function(_, title)
+	-- Don't change title if there is currently none
+	if not state.title then return end
```

Comment:
```lua
if state.title then update_title(title) end
```

Reply:
That was way too obvisous :disappointed:.